x-pack/plugins/observability/public/pages/overview/index.tsx

```diff
@@ -62,16 +58,22 @@ export const OverviewPage = ({ routeParams }: Props) => {
     return <LoadingObservability />;
   }
 
-  const {
-    rangeFrom = timePickerTime.from,
-    rangeTo = timePickerTime.to,
-    refreshInterval = 10000,
-    refreshPaused = true,
-  } = routeParams.query;
+  const { refreshInterval = 10000, refreshPaused = true } = routeParams.query;
 
-  const startTime = getParsedDate(rangeFrom);
-  const endTime = getParsedDate(rangeTo, { roundUp: true });
-  const bucketSize = calculatetBucketSize({ startTime, endTime });
+  const relativeTime = {
+    start: routeParams.query.rangeFrom ?? timePickerTime.from,
+    end: routeParams.query.rangeTo ?? timePickerTime.to,
+  };
+
+  const absoluteTime = {
+    start: moment.utc(getAbsoluteTime(relativeTime.start)).valueOf(),
+    end: moment.utc(getAbsoluteTime(relativeTime.end, { roundUp: true })).valueOf(),
+  };
```

Comment:
This should be enough:
```suggestion
  const absoluteTime = {
    start: getAbsoluteTime(relativeTime.start),
    end: getAbsoluteTime(relativeTime.end, { roundUp: true })
  };
```

`getAbsoluteTime` should look like this (note: `parsed.valueOf()`)

```tsx
export function getAbsoluteTime(rawDate?: string, opts = {}) {
  if (rawDate) {
    const parsed = datemath.parse(rawDate, opts);
    if (parsed) {
      return parsed.valueOf();
    }
  }
}
```

Reply:
`getAbsoluteTime` will always return `T | undefined` beacuse `parsed` might be undefined.